include/aspect/parameters.h

```diff
@@ -373,6 +373,50 @@ namespace aspect
       }
     };
 
+    /**
+     * A data structure containing a description of each field.
+     * At present, this structure only includes the field type
+     * (i.e., whether the field corresponds to a composition, porosity, etc.).
+     */
+    struct FieldDescription
+    {
+      /**
+       * This enum lists available field types.
+       */
+      enum Kind
+      {
+        composition,
```

Comment:
this should be updated to ``chemical_compostion``, right?

Reply:
Or just "chemical"?

Reply:
I think I would prefer chemical_composition, mostly because I associate it more with major element composition. I am thinking of how to classify fields like water content or trace elements, which we would not want to fall into the category of fields that are typically used to average density etc., but would clearly be 'chemical'. Is that just my personal opinion?

Reply:
I was just choosing the shorter option. I'm happy to go with your suggestion!